db/migrate/20160821085842_ext_management_system_atomic_type_to_openshift.rb

```diff
@@ -0,0 +1,16 @@
+class ExtManagementSystemAtomicTypeToOpenshift < ActiveRecord::Migration[5.0]
```

Comment:
I think for migrations that change data you have add specs - have a look a `spec/migrations` for examples